src/Controls/src/Core/ObservableWrapper.cs

```diff
@@ -106,7 +107,13 @@ IEnumerator IEnumerable.GetEnumerator()
 
 		public IEnumerator<TRestrict> GetEnumerator()
 		{
-			return _list.Where(i => i.Owned).OfType<TRestrict>().GetEnumerator();
+			foreach (TRestrict item in _list)
```

Comment:
Does this cause a cast exception when you have something in `_list` that isn't a `TRestrict`?

Reply:
I figured out a test that breaks, let me fix these.